examples/react-native/README.md

```diff
@@ -84,6 +84,32 @@ yarn react-native-example dev
 
 1. Open the app on the iOS simulator or Android emulator.
 
+## Storybook
+
+First boot Storybook Server, which will serve a web interface that allows switching between components (stories) and controlling multiple simulators/emulators:
+
+```bash
+# terminal 1
+yarn run react-native-example storybook
+```
+
+From a second terminal run ios simulator, android emulator or both:
+
+```bash
+# terminal 2
+
+# ios simulator
+yarn run react-native-example ios
```

Comment:
Since this command builds and installs the app on the iOS simulator, do we need to run this every time we run storybook? Same question for line 105

Reply:
No, just the first time, after which you just have to run metro, i'll update wording.